OpenRA.Mods.Common/Warheads/GrantExternalConditionWarhead.cs

```diff
@@ -30,6 +30,11 @@ public class GrantExternalConditionWarhead : Warhead
 		public override void DoImpact(in Target target, WarheadArgs args)
 		{
 			var firedBy = args.SourceActor;
+
+			if (target.Type == TargetType.Invalid) {
+				return;
+			}
```

Comment:
Please remove the braces:
```suggestion
			if (target.Type == TargetType.Invalid)
				return;
```